min-io/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-minio-service
+spec:
+  selector:
+    app: minio
+  ports:
+  - protocol: TCP
+    port: 80  # External access port
+    targetPort: {{ .Values.service.apiPort | default 9000 }}  # API port
+  - protocol: TCP
+    port: 9090  # Console access port
+    targetPort:  {{ .Values.service.consolePort | default 9090 }}
```

Comment:
Can you template the `port` fields as well instead of setting 80 and 9090? Also, you need to add names to those ports since you listed more than one